website/src/css/custom.css

```
@@ -278,6 +349,311 @@ html[data-theme='light'] code[class*='language-'] {
+  font-size: 1.08rem;
+  line-height: 1.75;
+}
+
+.theme-doc-markdown a:not(.button):not(.pagination-nav__link) {
+  text-decoration-color: rgba(95, 106, 252, 0.35);
+  text-decoration-thickness: 1px;
+  text-underline-offset: 0.18em;
+}
+
+.theme-doc-markdown a:not(.button):not(.pagination-nav__link):hover {
+  text-decoration-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)